api/controllers/ExportController.js

```diff
@@ -113,27 +117,41 @@ async function exportMembers(groupId, req, res) {
   }))
 
   // send data as CSV response
-  output(res, results, [
+  output(results, [
     'name', 'contact_email', 'contact_phone', 'location', 'avatar_url', 'tagline', 'bio',
     { key: 'url', header: 'personal_url' },
     'twitter_url', 'facebook_url', 'linkedin_url',
     'skills', 'skills_to_learn',
     'join_request_questions',
     'affiliations',
     'groups'
-  ])
+  ], email, groupId)
+
+
 }
 
 // toplevel output function for specific endpoints to complete with
-function output(res, data, columns) {
-  res.setHeader('Content-Type', 'text/csv')
-  res.setHeader('Content-Disposition', 'attachment; filename=\"' + 'download-' + Date.now() + '.csv\"')
-
+function output(data, columns, email, groupId) {
   stringify(data, {
     header: true,
     columns
+  }, (err, output) => {
+    const partialGroupId = groupId.slice(0,7)
+    const formattedDate = new Date().toISOString().slice(0,10)
+    const buff = Buffer.from(output)
+    const base64output = buff.toString('base64')
+
+    Queue.classMethod('Email', 'sendExportMembersList', {
+      email:  email,
+      files: [
+        {
+          id: `members-export-${partialGroupId}-${formattedDate}.csv`,
```

Comment:
we could use group name here instead of id

Reply:
Will address in next PR